_data/domains.yml

```diff
@@ -201,6 +201,12 @@ websites:
       tfa: No
       twitter: HeartInternet
 
+    - name: Hexonet
+      url: http://www.hexonet.net
```

Comment:
That URL directs to `https://www.hexonet.net/` (HTTPS). Please use that instead.